modules/gapi/test/gapi_gpu_test.cpp

```diff
@@ -202,6 +209,158 @@ TEST(GPU, Symm7x7_test)
         EXPECT_EQ(out_mat_gapi.size(), sz);
     }
 }
-#endif
+
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+void prepareD3D11Texture2D(const UINT width, const UINT height, std::vector<uint8_t>& test_data, ID3D11Texture2D** texture) {
+    std::iota(test_data.begin(), test_data.end(), uint8_t(0));
+    // Associate subresource with test data
+    D3D11_SUBRESOURCE_DATA initData = { test_data.data(),
+                                        sizeof(uint8_t) * width,
+                                        height };
+
+    // Create texture description
+    D3D11_TEXTURE2D_DESC desc = {};
+    desc.Width = width;
+    desc.Height = height;
+    desc.MipLevels = 1;
+    desc.ArraySize = 1;
+    desc.Format = DXGI_FORMAT_A8_UNORM; // CV_8U
+    desc.SampleDesc.Count = 1;
+    desc.Usage = D3D11_USAGE_DEFAULT;
+    desc.BindFlags = D3D11_BIND_SHADER_RESOURCE;
+    desc.MiscFlags = D3D11_RESOURCE_MISC_SHARED;
+    desc.CPUAccessFlags = 0; // 0 if CPU access is not required
+
+    HRESULT hr;
+    // Create device
+    ID3D11Device* mDevice = nullptr;
+    hr = D3D11CreateDevice(nullptr,
+                           D3D_DRIVER_TYPE_HARDWARE,
+                           nullptr,
+                           0,
+                           nullptr,
+                           0,
+                           D3D11_SDK_VERSION,
+                           &mDevice,
+                           nullptr,
+                           nullptr);
+
+    if(!SUCCEEDED(hr)) {
+        CV_Error(cv::Error::BadCallBack, "D3D11CreateDevice was failed: " +
+                    std::to_string(HRESULT_CODE(hr)));
```

Comment:
Test must be SKIPPED not FAILED if environment doesn't pass required expectations.

Reply:
Added skips